github_wrapper/requirements.txt

```diff
@@ -0,0 +1,2 @@
+requests==2.20.0
+flake8==2.6.2

```

Comment:
Why have you placed the requirements.txt file? 
If you are using pipenv utility, pipfile is the place to put package info

Reply:
I installed pipenv later for the project. This shouldn't be here.  Removing.